duckduckgo/iOS
# NetP Status View Header

Task/Issue URL: https://app.asana.com/0/0/1205084446087078/f

**Description**:

This is the first design implementation task in the overarching Network Protection Status View task and covers just the header view (that is, the views above the toggle view). I’m delaying on making this look good on iOS 14 for now as it will take some extra thought. But I am checking in with the Apple team and have created a follow-up task to fix iOS 14 at the end of the project if need be.

**Steps to test this PR**:
1. (If you haven’t followed the steps in [Network Protection iOS Smoke Tests: Invite Code flow](https://app.asana.com/0/0/1205201898139908) already, do so.)
2. **Check that the status view header (the views above, not including, the toggle view) matches [the designs](https://www.figma.com/file/zuCBX2UpJpS8CgtcmpU1hf/Network-Protection-for-iOS?type=design&node-id=252-21943&mode=design&t=Z7gEv95kIs3uMn7e-0) disconnected state** 
3. Switch the toggle to activate Network Protection.
4. **Check that the status view header again matches the designs but for the connected state**

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
